modules/tinymce/src/plugins/table/test/ts/browser/ui/TableDialogWithoutCssTest.ts

```diff
@@ -82,4 +82,16 @@ describe('browser.tinymce.plugins.table.TableDialogTest', () => {
     TableTestUtils.assertDialogValues(getExpectedData(2, '100%'), false, generalSelectors);
     await TableTestUtils.pClickDialogButton(editor, false);
   });
+
+  it('TINY-10308: Setting border to 0px should remove border-width styles and set border="0"', async () => {
+    const editor = hook.editor();
+    for (const border of [ '0', '0px' ]) {
+      editor.setContent('<table style="border-collapse: collapse;" border="1"><tbody><tr><td>&nbsp;</td></tr></tbody></table>');
```

Comment:
We should probably add a `border-width` style to one of the tds here and assert it is not removed. I am not sure if that is the ideal behaviour but is what happens on v5